cloud/azure/storage-account-capacity/variables.tf

```diff
@@ -0,0 +1,50 @@
+# Module specific
+
+# storage_capacity detectors
+
+variable "storage_capacity_disabled" {
+  description = "Disable all alerting rules for storage_capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "storage_capacity_disabled_critical" {
+  description = "Disable critical alerting rule for storage_capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "storage_capacity_disabled_major" {
+  description = "Disable major alerting rule for storage_capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "storage_capacity_notifications" {
+  description = "Notification recipients list per severity overridden for storage_capacity detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "storage_capacity_aggregation_function" {
+  description = "Aggregation function and group by for storage_capacity detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+}
+
+variable "storage_capacity_timer" {
```

Comment:
please use existing convention for lasting variables naming (and using seconds number): https://github.com/claranet/terraform-signalfx-detectors/blob/master/network/http/detectors-http.tf#L27